runtime/doc/treesitter.txt

```diff
@@ -225,15 +241,16 @@ repeated, for example, the following modeline blocks are all valid:
     ;; inherits: baz
 <
 
-Treesitter Query Predicates                         *lua-treesitter-predicates*
+==============================================================================
+TREESITTER QUERY PREDICATES                         *lua-treesitter-predicates*
 
-When writing queries for treesitter, one might use `predicates`, that is,
-special scheme nodes that are evaluated to verify things on a captured node
-for example, the |eq?| predicate : >
+Predicates are special scheme nodes that are evaluated to conditionally capture
+nodes. For example, the |eq?| predicate can be used as follows: >
     ((identifier) @foo (#eq? @foo "foo"))
 
-This will only match identifier corresponding to the `"foo"` text.
-Here is a list of built-in predicates :
+to only match identifier corresponding to the `"foo"` text.
+
+The following predicates are built in: (TODO: add missing predicates)
```

Comment:
@lewis6991 maybe you could help filling out the missing predicates?

Reply:
Nevermind, impatient bit me again and showed me the predicates from nvim-treesitter...

Reply:
How do you mean impatient bit you?

Reply:
I have nvim-treesitter as an optional package, but `packadd`ing it once adds it to the cache even if it is not explicitly loaded for the current session. This makes some of the extensions "hang around" permanently (until I clear the cache).